acceptance_tests/features/fixtures.py

```diff
@@ -69,7 +69,7 @@ def setup_data_with_internal_user_and_social_collection_exercise_to_closed_statu
 def setup_data_with_enrolled_respondent_user_and_internal_user(context):
     """ Creates default data + an enrolled Respondent in the default collection exercise """
     setup_data_with_internal_user(context)
-    create_enrolled_respondent_for_the_test_survey(context)
+    respondent = create_enrolled_respondent_for_the_test_survey(context)
```

Comment:
Why have you made this a variable?

Reply:
Old approach that I consolidate into the main `create_respondent` function